app/views/Settings/index.tsx

```diff
@@ -135,15 +135,6 @@ const SettingsScreen = ({ navigation }: SettingsScreenProps): JSX.Element => {
           </NativePicker>
         </View>
 
-        <View style={styles.section}>
```

Comment:
My understanding is that we wanted to allow users to get to the reporting a positive test flow from the settings screen, should we be removing this?

Reply:
If that's the case that's a BT only functionality -- on GPS the export flow is linked to as a main tab bar option. I was under the impression export is _not_ in BT right now?

Reply:
I might be a bit confused here, but this is a link to the "Affected User" flow which is different from the "Export my location data Flow" ?  the figma here: https://www.figma.com/file/79OwaKqHvSeriExpYzAvVN/PathCheck-%2F-Exploration-%2F-April%2C-May%2C-June-2020?node-id=3601%3A26330 , implies the Affected User flow is used by the BT app as well.

@tparesi , do you have any insight here?

Reply:
But yes, if this is just for the bt flow, we can definitely put it behind a !isGPS.

Reply:
hmm this might be a question for design. This should definitely be on the BT app on the settings screen as per the mocks John linked above. We can put it behind an !isGPS check for now and follow up with design.